CondFormats/SiPixelTransient/src/SiPixelUtils.cc

```diff
@@ -71,17 +71,95 @@ namespace SiPixelUtils {
     }
 
     //--- Finally, compute the position in this projection
-    float qdiff = q_l - q_f;
-    float qsum = q_l + q_f;
+    float q_diff = q_l - q_f;
+    float q_sum = q_l + q_f;
 
     //--- Temporary fix for clusters with both first and last pixel with charge = 0
-    if (qsum == 0)
-      qsum = 1.0f;
+    if (q_sum == 0)
+      q_sum = 1.0f;
 
-    //float hit_pos = geom_center + 0.5f*(qdiff/qsum) * w_eff + half_lorentz_shift;
-    float hit_pos = geom_center + 0.5f * (qdiff / qsum) * w_eff;
+    //float hit_pos = geom_center + 0.5f*(q_diff/q_sum) * w_eff + half_lorentz_shift;
```

Comment:
Please remove this line, or explain in which situation it is expected to be used